.circleci/config.yml

```diff
@@ -0,0 +1,253 @@
+version: 2.1
+executors:
+  default:
+    docker:
+    - image: circleci/golang:1.15.8
+
+aliases:
+- &restore_cache
+  restore_cache:
+    key: go-mod-v1-{{ checksum "go.sum" }}
+- &save_cache
+  save_cache:
+    key: go-mod-v1-{{ checksum "go.sum" }}
+    paths:
+      - "/go/pkg/mod"
+
+jobs:
+  check-style:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - *restore_cache
+    - run: make check-style
+    - *save_cache
+
+  check-modules:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - *restore_cache
+    - run: make check-modules
+    - *save_cache
+
+  check-boilerplate:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - run: ./scripts/verify-boilerplate.sh
+
+  check-mocks:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - run: make verify-mocks
+
+  test-sqlite:
+    executor:
+      name: default
+    steps:
+    - checkout
+    - *restore_cache
+    - run: |
+        make unittest
+        make goverall
+    - *save_cache
+
+  test-postgres:
+    docker:
+    - image: circleci/golang:1.14.6
+      environment:
+        GENESIS_DATABASE=postgres://genesis_test@localhost:5432/genesis_test?sslmode=disable
+    - image: circleci/postgres:11.2-alpine
+      environment:
+        POSTGRES_USER: genesis_test
+        POSTGRES_DB: genesis_test
+    executor:
+      name: default
+    steps:
+    - run:
+        name: Waiting for Postgres to be ready
+        command: |
+          for i in `seq 1 10`;
```

Comment:
just for consistency should we make this a bash script?

Reply:
or just use the [wait-for-it](https://github.com/vishnubob/wait-for-it)

Reply:
moved to its own script, thanks :)